.github/workflows/ci.yml

```diff
@@ -326,7 +326,9 @@ jobs:
         run: echo "EARTHLY_MAX_REMOTE_CACHE=true" >> $GITHUB_ENV
 
       - name: Configure Earthly to Push Artifacts
-        if: (github.ref == 'refs/head/master' || startsWith(github.ref, 'refs/head/release-')) && env.DOCKER_USR != '' && env.UPBOUND_MARKETPLACE_PUSH_ROBOT_USR != '' && env.AWS_USR != ''
+        if: |
+          (github.ref == 'refs/head/master' || startsWith(github.ref, 'refs/head/release-')) &&
+          (env.DOCKER_USR != '' && env.UPBOUND_MARKETPLACE_PUSH_ROBOT_USR != '' && env.AWS_USR != '')
```

Comment:
I'm confident `env.DOCKER_USR != '' && env.UPBOUND_MARKETPLACE_PUSH_ROBOT_USR != '' && env.AWS_USR != ''` evaluates to true, because it was before #5765. You can also see those env vars are not empty strings in the Actions output.

I'm also confident `github.ref == 'refs/head/master'` evaluates to true, because the above 'Configure Earthly to Push Cache..." step runs on master.

So given `A` is true and `X && Y && Z` is true I don't see why `(A || B) && X && Y && Z` wouldn't be true. 🤷‍♂️

Reply:
Aha. It's `refs/heads` not `refs/head`. 🙃